orttraining/orttraining/python/training/ortmodule/_custom_autograd_function_runner.py

```diff
@@ -164,6 +164,13 @@ def register_context(result):
 
             # Extract results as DLPack tensors plus autograd context. Also skips all None values.
             unwrapped_values = wrap_all_outputs(result, is_training_mode)
+            ctx = unwrapped_values[0]
+            if ctx and ctx.saved_tensors:
+                for tensor in ctx.saved_tensors:
+                    if tensor in set(new_wrapped_args):
+                        raise wrap_exception(
```

Comment:
I think you can just raise RuntimeError() here, and it will be caught in the line 169 and wrapped as another exception.

Reply:
thx, done.